Engine/Engine/Core/Application.cpp

```diff
@@ -88,8 +85,8 @@ void Application::Exit()
 	assetMan->SaveAssets();
 	Logger::Log(Logger::LogType::MSG, "Exiting engine");
 	std::string name = Logger::GetFormatedSystemTime();
-	name += "-log.txt";
-	assetMan->SaveAssetToFile("Log", name.c_str(), Logger::GetLog());
+	//name += "-log.txt";
+//	assetMan->SaveAssetToFile("Log", name.string().c_str(), Logger::GetLog());
```

Comment:
Just curious on why this is commented out. Is this code causing issues / crashes?